app/src/webview/java/org/mozilla/focus/webview/SystemWebView.java

```diff
@@ -109,6 +109,18 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
         WebViewProvider.applyAppSettings(getContext(), getSettings());
     }
 
+    @Override
+    public void onPause() {
+        super.onPause();
+        pauseTimers();
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        resumeTimers();
```

Comment:
This is super interesting. I haven't noticed resumeTimers()/pauseTimers() before. There's not much written about this - but it looks like it will pause JavaScript timers and this might be a good idea in Focus whenever the WebView is not in the foreground. Let's try that!